src/Parser/RestoDruid/SpellInfo.js

```diff
@@ -0,0 +1,245 @@
+  mastery: false,
+  masteryStack: false,
+  vers: true,
+};
+
+export const HEAL_INFO = {
+  [SPELLS.REJUVENATION.id]: {
+    int: true,
+    crit: true,
+    hasteHpm: true,
+    hasteHpct: true,
+    mastery: true,
+    masteryStack: true,
+    vers: true,
+  },
+  [SPELLS.REJUVENATION_GERMINATION.id]: {
+    int: true,
+    crit: true,
+    hasteHpm: true,
+    hasteHpct: true,
+    mastery: true,
+    masteryStack: true,
+    vers: true,
+  },
+  [SPELLS.REGROWTH.id]: { // TODO how handle direct vs HoT? Entries for direct for now.
+    int: true,
+    crit: true,
+    hasteHpm: false,
+    hasteHpct: true,
+    mastery: true,
+    masteryStack: true,
+    vers: true,
+  },
+  [SPELLS.WILD_GROWTH.id]: {
+    int: true,
+    crit: true,
+    hasteHpm: true,
+    hasteHpct: false,
+    mastery: true,
+    masteryStack: true,
+    vers: true,
+  },
+  [SPELLS.CULTIVATION.id]: {
+    int: true,
+    crit: true,
+    hasteHpm: true,
+    hasteHpct: true, // an approximation, because this is applied by Rejuv
+    mastery: true,
+    masteryStack: true,
+    vers: true,
+  },
+  [SPELLS.SPRING_BLOSSOMS.id]: {
+    int: true,
+    crit: true,
+    hasteHpm: true, // an approximation, because faster ticking Efflo will apply more
+    hasteHpct: false,
+    mastery: true,
+    masteryStack: true,
+    vers: true,
+  },
+  [SPELLS.CENARION_WARD.id]: {
+    int: true,
+    crit: true,
+    hasteHpm: true,
+    hasteHpct: false,
+    mastery: true,
+    masteryStack: true,
+    vers: true,
+  },
+  [SPELLS.FRENZIED_REGENERATION.id]: { // TODO does this actually scale with nothing?
+    int: false,
+    crit: false,
+    hasteHpm: false,
+    hasteHpct: false,
+    mastery: false,
+    masteryStack: true,
+    vers: false,
+  },
+  [SPELLS.LIFEBLOOM_HOT_HEAL.id]: {
+    int: true,
+    crit: true,
+    hasteHpm: true,
+    hasteHpct: false,
+    mastery: true,
+    masteryStack: true,
+    vers: true,
+  },
+  [SPELLS.LIFEBLOOM_BLOOM_HEAL.id]: {
+    int: true,
+    crit: true,
+    hasteHpm: false,
+    hasteHpct: false,
+    mastery: true,
+    masteryStack: false,
+    vers: true,
+  },
+  [SPELLS.HEALING_TOUCH.id]: {
+    int: true,
+    crit: true,
+    hasteHpm: false,
+    hasteHpct: true,
+    mastery: true,
+    masteryStack: false,
+    vers: true,
+  },
+  [SPELLS.SWIFTMEND.id]: {
+    int: true,
+    crit: true,
+    hasteHpm: false,
+    hasteHpct: false,
+    mastery: true,
+    masteryStack: false,
+    vers: true,
+  },
+  [SPELLS.LIVING_SEED.id]: { // an approximation based on what likely procced it. There is special case handling for crit.
+    int: true,
+    crit: true,
+    hasteHpm: false,
+    hasteHpct: true,
+    mastery: true,
+    masteryStack: false,
+    vers: true,
+  },
+  [SPELLS.TRANQUILITY_HEAL.id]: {
+    ignored: true, // Dreamgrove theorycrafters say should be ignored because it always overheals, could unfairly skew weights against Haste if included. I'd like a better solution, but this will do for now.
```

Comment:
I also have mixed feelings about it. I think I included it originally in my analyzer because I was trying to compare values with RDSW and TwigIt and they both make this decision too.

The concrete effect of ignoring Tranq is a higher value for Haste HPM, because it's the only stat that doesn't benefit Tranq.

I'm considering melding the ticks for each player into one big heal, and considering things then. So tick sets on any player where even one of the ticks overheals will be disregarded, but if a player takes all 5 ticks and isn't overhealed then I count it. I could think of a few similar tricks.